File: 1642-water-bottles/1642-water-bottles.cpp
class Solution {
public:
    int numWaterBottles(int numBottles, int numExchange) {
        int total=0;
        while(floor(numBottles/numExchange)){
            int rem=floor(numBottles/numExchange);
            int rem1=numBottles%numExchange;
            total+=(numBottles-rem1);
            numBottles=rem+rem1;
    } 
        total+=numBottles;
        return total;
        
    }
};

File: 1642-water-bottles/1642-water-bottles.py
class Solution:
    def numWaterBottles(self, numBottles: int, numExchange: int) -> int:
        total=0
        while(numBottles//numExchange):
            rem=numBottles//numExchange
            rem1=numBottles%numExchange
            total+=(numBottles-rem1)
            numBottles=rem+rem1
            
        total+=numBottles
        return total

        